packages/keystone/src/admin-ui/components/Navigation.tsx

```diff
@@ -73,9 +73,14 @@ const AuthenticatedItem = ({ item }: { item: { id: string; label: string } }) =>
         marginBottom: 0,
       }}
     >
-      <div css={{ fontSize: typography.fontSize.small }}>
-        Signed in as <strong>{item.label}</strong>
-      </div>
+      {item && item.state === 'authenticated' ? (
```

Comment:
I'd love to be able to pull this up into a single variable, but typescript can't infer the state otherwise so leaving it for now.